internal/remoteconfig/remoteconfig.go

```diff
@@ -71,7 +71,7 @@ const (
 	// APMTracingCustomTags enables APM client to set custom tags on all spans
 	APMTracingCustomTags
 	// APMTracingSampleRules represents the sampling rate using matching rules from APM client libraries
-	APMTracingSampleRules
+	APMTracingSampleRules = 29
```

Comment:
Maybe worth putting it in the constant block with APMTracingEnabled.

Reply:
done.